types/types.go

```diff
@@ -50,6 +51,16 @@ type CircuitBreaker struct {
 	Expression string `json:"expression,omitempty"`
 }
 
+// Buffering holds request/response buffering configuration/
+type Buffering struct {
+	Enabled              bool   `json:"enabled"`
```

Comment:
Can you delete the field `enabled`? Usually, in Træfik, defining attribute of a structure allows activating the feature.